.github/workflows/build-storybook-main.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Storybook Main
+
+on:
+  push:
+    branches:
+      - main
+  workflow_call:
+    secrets:
+      PUBLISH_STORYBOOK_TOKEN:
```

Comment:
This needs to be created. I will reach out to appropriate folks.

Reply:
@mcmire would you be able to create this `PUBLISH_STORYBOOK_TOKEN ` token or point it to this repo if it is still created for when it was used for the [design tokens repo](https://github.com/MetaMask/design-tokens/settings/secrets/actions) please?

Reply:
This has been added!

Reply:
Thank you!